<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-BPC4G7D" name="Module1" tests="2" failures="2" timestamp="19 May 2019 05:10:05 GMT" time="24.981" errors="0">
  <testcase name="f" time="0.188" classname="testnggg.Class1">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true],
	expected [10] but found [4]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true],
	expected [10] but found [4]
	at testnggg.Class1.f(Class1.java:26)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="0.136" classname="testnggg.Class2">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true],
	expected [10] but found [4]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true],
	expected [10] but found [4]
	at testnggg.Class2.f(Class2.java:26)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- Module1 -->
